core/o11y/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+use tracing_subscriber::EnvFilter;
+
+/// The default value for the `RUST_LOG` environment variable if one isn't specified otherwise.
+pub const DEFAULT_RUST_LOG: &'static str = "tokio_reactor=info,\
+     near=info,\
+     stats=info,\
+     telemetry=info,\
+     delay_detector=info,\
+     near-performance-metrics=info,\
+     near-rust-allocator-proxy=info,\
+     warn";
+
+/// Run the code with a default subscriber set to the option appropriate for the NEAR code.
+pub fn with_default_subscriber<R, F>(log_filter: EnvFilter, run: F) -> R
+where
+    F: FnOnce() -> R,
```

Comment:
I have a moderate preference towards guard-based API rather than closure-based API:

https://docs.rs/tracing/latest/tracing/subscriber/fn.set_default.html

As far as I understand, they are equivalent (closure seems to be implemented in terms of guard), but, syntactically, guards seems more light weight (smaller diff, no indents, etc)